tests/bsim/bluetooth/audio/src/bap_common.h

```diff
@@ -37,6 +37,10 @@
 
 #define LONG_META_LEN (sizeof((uint8_t []){LONG_META}) + 1U) /* Size of data + type */
 
+#define BROADCAST_CODE                                                                             \
+	((uint8_t[]){0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb, 0xcc, \
```

Comment:
Should we consider having a valid utf8 string?

Reply:
Broadcast codes are not requires to be UTF8/Ascii characters are they? 

Ideally they are for user input, but AFAIK it's not a requirement

Reply:
technically, I guess it could be whatever - but in practice, it should be something that is a valid UTF8 string, as I read this:

![image](https://github.com/zephyrproject-rtos/zephyr/assets/1014678/225569b5-4ee2-4f01-a172-23b4da3a2ce0)

Reply:
For added "realism", I think it could make sense to use a UTF8 string here too (even if the code works regardless)

Reply:
I somewhat agree. 

The section you quote is how it's represented on a UI level. 

While `0x00` is not a valid UTF8 value, it can be presentated as `"0x00"` by the UI. In this case we are testing a OS, and not a UI, so arguably `0x00, 0x01, 0x03` is as valid of a broadcast code as `"ABC"`. 

We did have an issue regarding the minimum broadcast code size as the minimum representation shall be 4 octets, but that requirement barely makes any sense:

1) `0x00` can represented as `"0x00"` and thus lives up to that requirement
2) "abcd" is 4 octets, but so is `🌮`